plugin-annotation/src/main/java/com/mapbox/mapboxsdk/plugins/annotation/Symbol.java

```diff
@@ -144,25 +146,26 @@ public void setIconRotate(Float value) {
   /**
    * Get the IconOffset property
    *
-   * @return property wrapper value around Float[]
+   * @return PointF value for Float[]
    */
-  public Float[] getIconOffset() {
+  public PointF getIconOffset() {
     JsonArray jsonArray = jsonObject.getAsJsonArray("icon-offset");
     Float[] value = new Float[jsonArray.size()];
```

Comment:
This also seems like an unnecessary object allocation, we could create `PointF` immediately.